tests/ci/run_fips_tests.sh

```diff
@@ -10,20 +10,17 @@ fips_build_and_test -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
 # The AL2 version of Clang does not have all of the required artifacts for address sanitizer, see P45594051
 if [[ "${AWSLC_NO_ASM_FIPS}" == "1" ]]; then
   if [[ ("$(uname -p)" == 'x86_64'*) ]]; then
-    echo "Building with Clang and testing AWS-LC in FIPS RelWithDebInfo mode with address sanitizer."
-    fips_build_and_test -DASAN=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_SHARED_LIBS=1
+    echo "Building with Clang and testing AWS-LC in FIPS Release mode with address sanitizer."
```

Comment:
Are these comments up-to-date? Are we building with clang? Do we also build with gcc? (Sorry I may have asked that before multiple times.

Reply:
> Are these comments up-to-date? 

Yes.

> Are we building with clang? 

Yes.

> Do we also build with gcc?

No when ASAN is used. We only use clang ASAN.